hybrid/decomposers.py

```diff
@@ -20,14 +20,17 @@
 from functools import partial
 from itertools import product
 from typing import Tuple
+import warnings
 
 import numpy as np
 import networkx as nx
+from networkx.algorithms.approximation.vertex_cover import min_weighted_vertex_cover
 
 import dimod
 from dimod.traversal import connected_components
 import dwave_networkx as dnx
 import dwave.preprocessing
+from dwave.samplers import SteepestDescentSolver, TreeDecompositionSolver
```

Comment:
@jackraymond, some tests are failing because `dwave.samplers` is not generally available for all versions of python/dimod.

For details see https://github.com/dwavesystems/dwave-hybrid/pull/284, but TL;DR: we need to drop support for `dimod<0.12` in order to switch to `dwave-samplers`/`dwave.samplers`.

Also, to even get access to Orang / `TreeDecompositionSolver` (only available as part of `dwave-samplers`) -- which is a pretty compelling reason.

I'll do that in a separate PR, and then we can rebase this PR.

Reply:
Lazy with my full environment testing!

Reply:
Done

Reply:
Thanks!